requirements.txt

```diff
@@ -2,3 +2,4 @@ torch
 tqdm
 numpy
 astunparse
+pytest
```

Comment:
I don't think `pytest` needs to be in here right?

Reply:
yeah true, i was testing there and forgot to remove it.